swerve/TorqueSwerveX.java

```diff
@@ -16,7 +17,6 @@
 import edu.wpi.first.math.kinematics.SwerveModulePosition;
 import edu.wpi.first.math.kinematics.SwerveModuleState;
 import edu.wpi.first.wpilibj.DriverStation;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 /**
  * West Coast Products Swerve X Module.
```

Comment:
Make the name `TorqueSwerveModuleX` for consistency 

This lib is public--another team reached out to me yesterday abt using our swerve. Our code for these modules must be good

Reply:
ok